docs/stencil-docs/deploying-a-theme/theme-best-practices.md

```diff
@@ -0,0 +1,169 @@
+
+## Injecting variables
+
+Injecting javascript context variables allow you to access store data through your theme. You can inject javascript context variables using the `{{inject}}` helper for properties you want responses for by assigning a custom variable name. You can then use the `{{jsContext}}` handlebars helper to access a stringified JSON object containing all injected data with your assigned variable name as the key in key-value pairs. For more information, see [injection helpers](https://developer.bigcommerce.com/stencil-docs/reference-docs/handlebars-helpers-reference#injection-helpers). The code example below uses `{{inject}}` and `{{jsContent}}` to log the product name to your browser's console when added to Cornerstone's `product.html` file: 
+
+```javascript
+{{inject "myProductName" product.title}}
+
+<script>
+// Note the lack of quotes around the jsContext handlebars helper, it becomes a string automatically.
+var jsContext = JSON.parse({{jsContext}});
+/* jsContext would output "{\"myProductName\": \"Sample Product\"}" which can feed directly into
+your JavaScript. */
+console.log(jsContext.myProductName); // Will output: Sample Product
+</script>
+```
+
+We suggest injecting only specific variables and only the variables you need. If you inject all settings, you could accidently inject PII (Personally-Identifying Information) and cause performance issues with your site.
+
+## Using Lighthouse
+
+Google [Lighthouse](https://developers.google.com/web/tools/lighthouse) is an open-source tool used to improve the performance, quality, accessibility, best practices, and more for web pages. Enter a URL into Lighthouse, and it will generate a series of audits against the page. We highly encourage you to use Lighthouse as a validation step.
+
+We recommend that you use Lighthouse to focus on the following metrics for optimizing your BigCommerce store.
+* [Largest Contentful Paint (LCP)](#largest-contentful-paint)
+* [First Input Delay (FID)](#first-input-delay)
+* [Cumulative Layout Shift (CLS)](#cumulative-layout-shift)
+
+### Largest Contentful Paint
+LCP is the metric that reports the time it takes to display the largest image on the screen. This metric is important because page speed ensures visitors receive the best performance and presentation of the site.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+**Good score:**
+The optimal LCP score is 2.5 seconds or faster. 
+
+</div>
+</div>
+</div>
+
+To improve LCP:
```

Comment:
Be consistent with the lack of punctuation at the end of bulleted items on lists